connectivity/check/context.go

```diff
@@ -48,6 +51,19 @@ type ConnectivityTest struct {
 	lastFlowTimestamps map[string]time.Time
 }
 
+type PerfTests struct {
+	Pod  string
+	Test string
+}
+
+type PerfResult struct {
+	Metric   string
+	Duration int
```

Comment:
Let's change this to a `time.Duration` as well (and change the format specifier in `ct.Logf` from `%-15d` to `%-15s` so printing the result will show a nicely formatted duration, e.g.

```
🔥 Performance Test Summary
-----------------------------------------------------------------------------------------------------------------------------
📋 Scenario                                           | Test            | Num Samples     | Duration        | Avg value      
-----------------------------------------------------------------------------------------------------------------------------
📋 perf-client-other-node-674f5b4c96-bxbsq            | TCP_RR          | 1               | 1s              | 9088.24 (OP/s)
📋 perf-client-other-node-674f5b4c96-bxbsq            | TCP_CRR         | 1               | 1s              | 2779.66 (OP/s)
📋 perf-client-56cf954b5c-hgq8x                       | TCP_RR          | 1               | 1s              | 26610.28 (OP/s)
📋 perf-client-56cf954b5c-hgq8x                       | UDP_STREAM      | 1               | 1s              | 689.69 (Mb/s)
📋 perf-client-56cf954b5c-hgq8x                       | UDP_RR          | 1               | 1s              | 34702.63 (OP/s)
📋 perf-client-other-node-674f5b4c96-bxbsq            | TCP_STREAM      | 1               | 1s              | 887.87 (Mb/s)
📋 perf-client-other-node-674f5b4c96-bxbsq            | UDP_RR          | 1               | 1s              | 10092.40 (OP/s)
📋 perf-client-other-node-674f5b4c96-bxbsq            | UDP_STREAM      | 1               | 1s              | 1048.76 (Mb/s)
📋 perf-client-56cf954b5c-hgq8x                       | TCP_CRR         | 1               | 1s              | 7271.43 (OP/s)
📋 perf-client-56cf954b5c-hgq8x                       | TCP_STREAM      | 1               | 1s              | 666.74 (Mb/s)
-----------------------------------------------------------------------------------------------------------------------------
```